android/src/main/java/io/grpc/android/AndroidChannelBuilder.java

```diff
@@ -138,20 +126,6 @@ public AndroidChannelBuilder sslSocketFactory(SSLSocketFactory factory) {
     }
   }
 
-  /** Set the delegate channel builder's scheduledExecutorService. */
```

Comment:
I expect more people need this. The only reason they may not now is because it was unavailable for so long.